index.html

```diff
@@ -0,0 +1,123 @@
+        <input
+        name="email"
+        id="email"
+        type="email"
+        placeholder="Enter your email address"
+        required
+      />
+      <input id="submit" type="submit" value="Get Started" class="btn" />
+      </form>
+    </section>
+    
+    
+      <section id="home">
+        <div class="row">
+          <div class="icon"><i class="fas fa-chart-line"></i></div>
+          <div class="body">
+            <h2>Improve your Company!</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At asperiores quae qui enim eos dolor eaque repudiandae, facere odio, similique, excepturi libero distinctio architecto sapiente harum expedita nulla. Nemo, unde.</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="icon"><i class="fas fa-globe-americas"></i></div>
+          <div class="body">
+            <h2>Reach all the world with one clic!</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At asperiores quae qui enim eos dolor eaque repudiandae, facere odio, similique, excepturi libero distinctio architecto sapiente harum expedita nulla. Nemo, unde.</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="icon"><i class="fas fa-address-card"></i></div>
+          <div class="body">
+            <h2>Share your Resume!</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. At asperiores quae qui enim eos dolor eaque repudiandae, facere odio, similique, excepturi libero distinctio architecto sapiente harum expedita nulla. Nemo, unde.</p>
+          </div>
+        </div>
+      </section>
+    
+
+    <section id="about">
+      <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/Fn2OfRGbxwc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
+
+    <section id="pricing">
+      <div class="product">
+        <h2>$1600</h2>
+        <ol>
+          <li>Lorem</li>
+          <li>Lorem</li>
+          <li>Lorem</li>
+          <li>Lorem</li>
+        </ol>
+        <button class="btn">
+          Select
+        </button>
+      </div>
+      <div class="product">
+        <h2>$3200</h2>
+        <ol>
+          <li>Lorem</li>
+          <li>Lorem</li>
+          <li>Lorem</li>
+          <li>Lorem</li>
+        </ol>
+        <button class="btn">
+          Select
+        </button>
+      </div>
+      <div class="product">
+        <h2>$7400</h2>
+        <ol>
+          <li>Lorem</li>
+          <li>Lorem</li>
+          <li>Lorem</li>
+          <li>Lorem</li>
+        </ol>
+        <button class="btn">
+          Select
+        </button>
+      </div>
+    </section>
+    <footer>
+      <span>Copyright 2022 - Tom&Emannuel</span>
+    </footer>
+  </div>
+  </div>
```

Comment:
In this part, there is an extra div container that you can remove because you already have all the code contained inside the body tag.
Please change that part of the code so you can have a clean code.
Thanks 😃